molecule/logger.py

```diff
@@ -19,11 +19,27 @@
 #  DEALINGS IN THE SOFTWARE.
 
 import logging
+import os
 import sys
 
 import colorama
+from ansible.module_utils.parsing.convert_bool import boolean as to_bool
 
-colorama.init(autoreset=True)
+
+def should_do_markup(fh=sys.stdout):
```

Comment:
Why do you need this to be an argument? I see that it's good for unit-testing but you don't really test the integration then.

Reply:
That is unit testing, I cannot really control how the user is calling the unittests and I need to isolate that from the test (it could be inside/outside tox, in CI, command line, with output redirected to file,... all of these could influence what stdout.istty() returns).